addons/houseLightSwitch/functions/fnc_ensureFuseboxHelper.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+
+params [
+	["_building", objNull, [objNull]]
+];
+assert(!(isNull _building));
+	
+private _helper = _building getVariable [QGVAR(fuseboxHelper), objNull];
+if (_helper isNotEqualTo objNull) exitWith {
+	TRACE_1("helper for %1 exists already!", _building);
+};
+
+private _helperPos = _building modelToWorld [0, 0, 0];
```

Comment:
add more smart here to create helper somewhere reachable.

nomi-approach : get first position within building, then move Z coords of helperpos to that + 1.7